ddlambda.go

```diff
@@ -221,7 +221,7 @@ func (cfg *Config) toTraceConfig() trace.Config {
 	}
 
 	if tracingEnabled, err := strconv.ParseBool(os.Getenv(DatadogTraceEnabledEnvVar)); err == nil {
-		traceConfig.DDTraceEnabled = tracingEnabled;
+		traceConfig.DDTraceEnabled = tracingEnabled
```

Comment:
Is there anything special of us removing this `;`?

Reply:
`go fmt` was run on the repo

Reply:
Tysm